src/main/resources/config/defaults-storage-pravega.json

```diff
@@ -31,26 +33,15 @@
 			"sndBuf": 0,
 			"tcpNoDelay": true,
 			"timeoutMillis": 0,
-			// The flag to enable the load through SSL/TLS.
```

Comment:
Why did you delete the comments?

Reply:
Error in json format during compilation locally
Travis doesn't check it?